site-src/geps/gep-713.md

```diff
@@ -38,10 +48,116 @@ portable.
+augment the behavior of the bound Service object with the structured data in the
+Spec of BackendCapabilities, since it’s not easy to make changes to a GA upstream
+object like Service. (And arguably, Service is already a pretty overloaded
+resource, so there will need to be good justification for any changes).
+
+With these two examples in mind, here are some guidelines for when to consider
+using a targeted metaresource:
+* The number or scope of objects to be modified is limited or singular. Targeted
+  metaresources should target one specific object (preferred), or a tightly-scoped
+  set of objects (like ReferenceGrant).
+* The modifications to be made to the objects don’t have any transitive information -
+  that is, the modifications only affect the single object that the targeted
+  metaresource is bound to, and don’t have ramifications that flow beyond that
+  object. ReferenceGrant and BackendCapabilities are good examples here because
+  they don’t expect the metaresource to modify the behavior of more than one
+  object. (Even in the ReferenceGrant case, the ability to set a Kind rather
+  than an individual object is syntactic sugar for having to prevent needing to
+  have lots of ReferenceGrants if you have lots of wrapped objects).
+* In terms of status, it should be reasonably easy for a user to understand that
+  everything is working - basically, as long as the targeted object exists, and
+  the modifications are valid, the metaresource is valid, and this should be
+  straightforward to communicate in one or two Conditions. Note that at the time
+  of writing, this is *not* completed.
+* Be extremely careful in using metaresources to target things in other namespaces.
+  In general, this is a big security problem, unless the referent resources have
+  some way to decline. Most of the time, it’s better to have the resource target
+  something in its own namespace (that is, the targeted resource describes what
+  should have its behavior modified), with other, similar but distinct fields
+  then describe what to target for further modifications. An example:
+  * A metaresource that constrains client behavior when reaching out to other
+    services should use a targetRef-like field to choose the clients, and have
+    another field that sets what the external targets of the constrained behavior
+    are. (The external targets may conceivably be “all external targets”).
+  In general, metaresources that modify the behavior of things outside their own
+  namespace should be avoided unless it uses a handshake of some sort, where the
+  things outside the namespace can opt–out of the behavior. (Notably, this is the
+  design that we used for ReferenceGrant).)
+
+### Policy Attachment: It's all about the defaults and overrides
+
+Because a Policy is a metaresource, it targets some other resource and _augments_
+its behavior.
+
+But why have this distinct from other types of metaresource? Because Policy
```

Comment:
In my opinion, there are still valid cases of policy attachment that could look more like a "Targeted Metaresource" (or "Direct Policy Attachment"). In GKE, we're using HealthCheckPolicy to configure custom health checks, but of course this can only be attached to a Service, so the direct attachment model would have actually been useful for us in this case. https://cloud.google.com/kubernetes-engine/docs/how-to/configure-gateway-resources#configure_health_check.

Reply:
Updated.